SampleApps/TestApp/ContentView.swift

```diff
@@ -258,40 +261,40 @@ struct CustomIdentifierView: View {
                     Text("Remove Identity")
                 }.padding()
             }
-
+            
         }
     }
 }
 
 struct MultipleIdentityView: View {
     private let edgeIdentityStoredDataKey = "Adobe.com.adobe.edge.identity.identity.properties"
     private let identityStoredDataKey = "Adobe.com.adobe.module.identity.identity.properties"
-
+    
     @ObservedObject var extensions: RegisteredExtensions
     @State var ecidEdgeIdentityText: String = ""
     @State var ecidIdentityText: String = ""
     @State var identityMapText: String = ""
-
+    
     var body: some View {
         VStack {
             HStack(alignment: .center) {
                 Image(systemName: extensions.isEdgeIdentityRegistered ? "circle.fill" : "circle")
                     .foregroundColor(Color.blue)
-
+                
                 Button(action: {
                     if extensions.isEdgeIdentityRegistered {
                         MobileCore.unregisterExtension(AEPEdgeIdentity.Identity.self)
                     } else {
                         MobileCore.registerExtension(AEPEdgeIdentity.Identity.self)
                     }
-
+                    
```

Comment:
Can you run `make format` or are these changes after formatting?

Reply:
Thank you, forgot to run the autocorrect! Should be applied with the latest commit